docs/build/references/command-line-interface.md

```diff
@@ -681,6 +684,22 @@ If true, database pruning of obsolete historical data will be enabled. Should be
 
 If true, enables preimages. Defaults to false.
 
+#### `offline-pruning-enabled` (boolean):
+
+If true, offline pruning will run on startup and block until it completes (approximately one hour on mainnet). This will reduce the size of the database by deleting old trie nodes.
+
+Since offline pruning deletes old state data, this should not be run on nodes that need to support archival API requests.
+
+This is meant to be run manually, so after running with this flag once, it must be toggled back to false before running the node again. Therefore, you should run with this flag set to true and then set it to false on the subsequent run.
+
+#### `offline-pruning-bloom-filter-size` (int):
+
+This flag sets the size of the bloom filter to use in offline pruning (denominated in MB and defaulting to 512 MB).
```

Comment:
Can we make this like:
`This flag sets the size of the in memory bloom filter to use in offline pruning (denominated in MB and defaults to the recommended setting of 512 for mainnet use).`

Reply:
I don't think we want to refer to it strictly as in-memory bloom filter, since we do write it to disk as well, but I'll rephrase this.